backend/data/src/main/kotlin/io/tolgee/util/WordCounter.kt

```diff
@@ -4,23 +4,26 @@ import com.ibm.icu.text.BreakIterator
 import com.ibm.icu.util.ULocale
 
 object WordCounter {
+  private val NON_WORD = """[\p{P} \t\n\r~!@#$%^&*()_+{}\[\]:;,.<>/?-]""".toRegex()
+  private val LANGUAGE_PART = "^([A-Za-z]+).*".toRegex()
+
   fun countWords(text: String, languageTag: String): Int {
     val uLocale = getLocaleFromTag(languageTag)
     val iterator: BreakIterator = BreakIterator.getWordInstance(uLocale)
     iterator.setText(text)
 
-    val words: MutableList<String> = ArrayList()
+    var words = 0
```

Comment:
Oh, thanks. I used the list for testing and i probably forgot to make it counter only.